sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece-response.integration.service.ts

```diff
@@ -30,9 +30,7 @@ export class ECEResponseIntegrationService extends SFTPIntegrationBase<
   ): Promise<ECEResponseFileDetail[]> {
     let fileLines: false | string[];
```

Comment:
`let fileLines: string[];` ?

Reply:
Also this piece of code can be removed. 
![image](https://github.com/user-attachments/assets/fbc54570-5008-4302-a1cd-3fb6bff3c39b)